src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -129,7 +129,7 @@ internal class HttpSource
             {
                 HttpStatusCode statusCode;
 
-                using (var response = await _client.SendAsync(request))
+                using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
                 {
                     response.EnsureSuccessStatusCode();
```

Comment:
If we could avoid making the file at all in that case it would be better but not if the code gets nasty :smile: